keylime/tenant.py

```diff
@@ -589,6 +591,8 @@ def do_cvstatus(self):
 
         response = do_cvstatus.get((f"/v{self.api_version}/agents/{self.agent_uuid}"), timeout=self.request_timeout)
 
+        response_json = Tenant._jsonify_response(response)
+
```

Comment:
there are two more `return response.json()` at lines 602 and 605 that probably should also be converted just like you converted line 613.

Reply:
Same